### PR TITLE
Feature/mtm 40362 core properties for configuring lower upper interval

### PR DESCRIPTION
**Pull request by Markus-SWAG**
With MTM-40362 the manifest entry "requestedResources" can be over-written based on system settings. For "resources" and "requestedResources" the defaults can be set by system settings by an administrator. The manifest description reflects now the changes.

### CODE REVIEW

File: content/microservice-sdk/concept-bundle/manifest.md [hunk 1]
```diff
@@ -210,17 +211,21 @@ The snapshot postfix means that the image build is a snapshot of your applicatio
...
-|cpu|String |Limit for number of CPUs or CPU time <br>Default CPU: 0.5, min: 0.1<br>Default CPU time: 500m, min: 100m | No
-|memory|String |Limit for microservice memory usage <br>Default: 512M, Min: 10M<br/>Possible units are: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki |No
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+|      |      |             |          |
+| :--- | :--- | :---------- | :---------- |
+| ---- | ---- | ----------- | ----------- |
+|      |      |             |             |
+|cpu|String |Limit for number of CPUs or CPU time <br>Default CPU: 0.5, min: 0.1 <br>Default CPU time: 500m, min: 100m <br>There may be a different system setting for the Default CPU by administrator.| No
+|memory|String |Limit for microservice memory usage <br>Default: 512M, Min: 10M<br/>Possible units are: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki <br>There  may be a different system setting for the Default by administrator.|No
```
Comment (rpr-github) [inline content/microservice-sdk/concept-bundle/manifest.md:222]: Proposal: A different default value may be configured by the system administrator.

File: content/microservice-sdk/concept-bundle/manifest.md [hunk 2]
```diff
@@ -210,17 +211,21 @@ The snapshot postfix means that the image build is a snapshot of your applicatio
...
 
-|Name|Type|Description|Required|
-|:---|:---|:----------|:----------|
-|cpu|String |Limit for number of CPUs or CPU time <br>Default CPU: 0.5, min: 0.1<br>Default CPU time: 500m, min: 100m | No
-|memory|String |Limit for microservice memory usage <br>Default: 512M, Min: 10M<br/>Possible units are: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki |No
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+|      |      |             |          |
+| :--- | :--- | :---------- | :---------- |
+| ---- | ---- | ----------- | ----------- |
```
Comment (rpr-github) [inline content/microservice-sdk/concept-bundle/manifest.md:218]: The lines with many "|" and "-" characters appear a bit strange.

File: content/microservice-sdk/concept-bundle/manifest.md [hunk 3]
```diff
@@ -210,17 +211,21 @@ The snapshot postfix means that the image build is a snapshot of your applicatio
...
+|memory|String |Limit for microservice memory usage <br>Default: 512M, Min: 10M<br/>Possible units are: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki <br>There  may be a different system setting for the Default by administrator.|No
 
 #### RequestedResources
 
 |Name|Type|Description|Required|
 |:---|:---|:----------|:----------|
-|cpu|String |Minimal requirements for number of CPUs or CPU time  <br>Default: 250m|No
-|memory|String |Minimal requirements for microservice memory usage <br>Default: 256M <br/>Possible postfix values are: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki |No
+|cpu|String |Intended minimal requirements for number of CPUs or CPU time  <br>The value may be over-written based on system settings. <br>Default: 250m<br>There  may be a different system setting for the Default by administrator.|No
+|memory|String |Intended minimal requirements for microservice memory usage <br>The value may be over-written based on system settings. <br>Default: 256M <br/>Possible postfix values are: E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki<br>There  may be a different system setting for the Default by administrator. |No
```
Comment (rpr-github) [inline content/microservice-sdk/concept-bundle/manifest.md:228]: see above

File: content/microservice-sdk/concept-bundle/manifest.md [hunk 4]
```diff
@@ -137,13 +137,13 @@ See below for detailed information about available settings.
...
 <td style="text-align:left">Resources</td>
-<td style="text-align:left">Configuration for resources limits.<br>Guaranteed resources are CPU=0.25, Memory=256MB<br>Default limits are CPU=0.5, Memory=512MB</td>
+<td style="text-align:left">Configuration for resources limits.<br>Default limits are CPU=0.5, Memory=512MB. There may be other defaults set by an administrator.</td>
 <td style="text-align:left">No</td>
 </tr>
 <tr>
 <td style="text-align:left">requestedResources</td>
 <td style="text-align:left">RequestedResources</td>
-<td style="text-align:left">Configuration for minimal required resources.<br>Default values are CPU=0.25, Memory=256MB</td>
+<td style="text-align:left">Intended configuration for minimal required resources.<br>The value may be over-written based on system settings.<br>Default values are CPU=0.25, Memory=256MB. There may be other defaults set by an administrator.</td>
```
Comment (rpr-github) [inline content/microservice-sdk/concept-bundle/manifest.md:146]: "The value**s** may be over-written ..."

File: content/microservice-sdk/concept-bundle/manifest.md (hunk 2 above)
Comment (Markus-SWAG) [reply content/microservice-sdk/concept-bundle/manifest.md:218]: Never use a WYSIWYG-Editor...

File: content/microservice-sdk/concept-bundle/manifest.md (hunk 1 above)
Comment (Markus-SWAG) [reply content/microservice-sdk/concept-bundle/manifest.md:218]: done.

**Review by rssag — APPROVED** [review]

**Review by rpr-github — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by manasesjesus — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Markus-SWAG Is this only supposed to go into 10.12 and later or is this already relevant for earlier releases?

**Comment by Markus-SWAG** [comment]
> @Markus-SWAG Is this only supposed to go into 10.12 and later or is this already relevant for earlier releases?

It is only going to 10.12 or later.
@BeateRixen: The actual merge will be done by someone from docu?

**Comment by BeateRixen** [comment]
Yes, done.

**Pull request merged**